controller/uninstall_controller.go

```diff
@@ -617,6 +622,23 @@ func (c *UninstallController) deleteReplicas(replicas map[string]*longhorn.Repli
 	return
 }
 
+// deleteLeftBackups deletes the backup having no backup volume
+func (c *UninstallController) deleteLeftBackups(backup *longhorn.Backup) (err error) {
+	backupVolumeName, ok := backup.Labels[types.LonghornLabelBackupVolume]
+	if !ok {
+		// directly delete it if there is even no backup volume label
+		c.ds.DeleteBackup(backup.Name)
```

Comment:
What happens if this operation fails?

Reply:
Also need to catch the error here, thanks for the remind!

Reply:
Fixed.